preferences/src/main/java/dev/jdtech/jellyfin/AppPreferences.kt

```diff
@@ -22,8 +22,9 @@ constructor(
         }
 
     // Appearance
-    val theme get() = sharedPreferences.getString(Constants.PREF_THEME, null)
+    val theme: String? get() = sharedPreferences.getString(Constants.PREF_THEME, null)
```

Comment:
I would not add `: String?` as the type is already inferred from the getter.
This was probably added during testing?

Reply:
yeap, I think I already removed it, and after running the test, it appears automatically again.